tests/modules/gatk4/genotypegvcfs/main.nf

```diff
@@ -129,16 +134,14 @@ workflow test_gatk4_genotypegvcfs_gendb_input_intervals {
     fastaIndex   = file(params.test_data['homo_sapiens']['genome']['genome_fasta_fai'], checkIfExists: true)
     fastaDict    = file(params.test_data['homo_sapiens']['genome']['genome_dict'], checkIfExists: true)
 
-    intervalsBed = file(params.test_data['homo_sapiens']['genome']['genome_bed'], checkIfExists: true)
-
     test_genomicsdb = file(params.test_data['homo_sapiens']['illumina']['test_genomicsdb_tar_gz'], checkIfExists: true)
 
     UNTAR ( test_genomicsdb )
     gendb = UNTAR.out.untar.collect()
     gendb.add([])
-    input = Channel.of([ id:'test' ]).combine(gendb)
+    input = Channel.of([ id:'test' ]).combine(gendb).add(file(params.test_data['homo_sapiens']['genome']['genome_bed'], checkIfExists: true))
```

Comment:
Code doesn't seem to like when .combine() and .add() are mixed. might need to do gendb.add([]) and then gendb.add([path/to/genome_bed]) before performing the combine